src/ts/toolbar.ts

```diff
@@ -70,8 +70,13 @@ document.addEventListener('DOMContentLoaded', () => {
         if (['/team.php', '/ucp.php'].includes(window.location.pathname) && !urlParams.has('templateid')) {
           window.location.replace('?tab=3');
         } else {
-          document.getElementById('toggle-pin-icon').classList.toggle('color-weak');
-          ['bgnd-gray', 'hl-hover-gray'].forEach(cl => document.getElementById('toggle-pin-icon-div').classList.toggle(cl));
+          const container = document.getElementById('container');
```

Comment:
is there a reason you're not using the `document` here? Meaning do `document.querySelector` instead of targeting the container.

Reply:
So it turns out that we really don't need to do that at all. See https://github.com/elabftw/elabftw/pull/4623/commits/e23ad897844e1645f9a96101c17a08489b07ec8f

Reply:
> is there a reason you're not using the `document` here? Meaning do `document.querySelector` instead of targeting the container.

It limits the search space a bit, so it should have a better performance. Ideally, it would be even more specific.

Reply:
> It limits the search space a bit, so it should have a better performance

Yeah, but it adds code and I actually just measured the perf. If you do it 10.000 times, you can save 4 milliseconds using the container instead of document (80 ms -> 76 ms). Meaning the perf gain isn't something we can measure when doing it once because it's probably something in microseconds (less than a μs even).

Anyway, now it's much better because we reuse the `el` that was clicked!

Keeping in mind perfs is indeed important, but sometimes the gain is so small that it doesn't justify making the code more complex! ;)